src/core/InputDropdown/style.ts

```diff
@@ -115,37 +122,21 @@ const minimal = (props: InputDropdownProps): SerializedStyles => {
     }
 
     &:hover {
-      color: ${colors?.gray[600]};
+      background-color: ${colors?.gray[100]};
       border: none;
-
-      span {
-        color: ${colors?.gray[600]};
-      }
-
-      path {
-        fill: ${colors?.gray[600]};
-      }
     }
 
     &:active {
-      color: ${palette?.text?.primary};
+      background-color: ${colors?.gray[100]};
       border: none;
-
-      span {
-        color: #000;
-      }
-
-      path {
-        fill: ${colors?.primary[400]};
-      }
     }
 
     &:focus {
       outline: none;
     }
 
     &.MuiButton-root.MuiButton-text > span {
-      ${fontHeaderS(props)}
+      ${labelFontBodyS(props)}
```

Comment:
@masoudmanson We changed to `labelFontBodyS` because the Figma design file for the new minimal InputDropdown says to use fontBodyS